pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -1962,8 +1962,18 @@ class Build {
                                 repoHandler.checkoutUserBuild(context)
                                 printGitRepoInfo()
                                 if (buildConfig.VARIANT == "openj9") {
-                                    context.sshagent(['83181e25-eea4-4f55-8b3e-e79615733226']) {
-                                        context.sh(script: "./${DEFAULTS_JSON['scriptDirectories']['buildfarm']}")
+                                    def buildArgs
+                                    if (DEFAULTS_JSON['bundle-openjceplus'] == true) {
+                                        buildArgs = ' --bundle-openjceplus'
+                                    } else {
```

Comment:
We can eliminate the `else` if we change the def to `def buildArgs = ''`.

Reply:
Changed.